src/NLog.Extensions.Logging/LogEventInfoExtensions.cs

```diff
@@ -0,0 +1,36 @@
+using Microsoft.Extensions.Logging;
+
+namespace NLog.Extensions.Logging
+{
+
+    public static class LogEventInfoExtensions {
+
+
+        /// <summary>
+        /// Gets the <see cref="Microsoft.Extensions.Logging.EventId"/>. If it was not included in the log 
+        /// event, then uses the default value.
+        /// </summary>
+        /// <param name="logEvent"></param>
+        /// <param name="defaultValue"></param>
+        /// <returns></returns>
+        public static EventId getEventId(this LogEventInfo logEvent, EventId defaultValue = default(EventId))
```

Comment:
we could return a nullableEventId? That saves use the 2nd parameter.

Also check the naming, we follow the default naming rules (so `GetEventId`